code/modules/client/preferences_savefile.dm

```diff
@@ -68,7 +68,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 					if(8)
 						be_special += ROLE_WIZARD
 					if(16)
-						be_special += ROLE_MALF
+						be_special += ROLE_REVENANT

```

Comment:
No, this is not at all what this is for.
These values can NOT be changed ad-hoc later on.
If 16 (malf) no longer does anything, have it have an empty if and say so, leave revenant where it is.

This is to convert old savefiles to the new format, you can't change the past Kor.